components/molecule/select/test/index.test.js

```diff
@@ -100,11 +100,11 @@ describe(json.name, () => {
       const {getByRole} = setup(props)
       const textBox = getByRole('textbox')
 
-      userEvents.click(textBox)
-      userEvents.tab()
+      fireEvent.focusIn(textBox)
+      fireEvent.focusOut(textBox)
 
       // Then
-      sinon.assert.calledOnce(spy)
+      sinon.assert.called(spy)
```

Comment:
isn't `calledOnce` a better choice for this kind of test? being it more specific, the test is more precise, IMHO. What do you think?

Reply:
Good catch! will update this. Thanks! (: